node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -85,6 +102,10 @@ impl ChainScraper {
 	/// As long as we have `MAX_FINALITY_LAG` this makes sense as a value.
 	pub(crate) const ANCESTRY_SIZE_LIMIT: u32 = MAX_FINALITY_LAG;
 
+	/// How many blocks after finalization a backed and not included candidate should
+	/// be kept.
+	pub(crate) const BACKED_CANDIDATE_MAX_LIFETIME: BlockNumber = 2;
```

Comment:
Why do we special case backed candidates here? Shouldn't the same reasoning apply to included candidates?